psalm-report.json

```diff
@@ -0,0 +1,5 @@
+{
+    "error": [],
+    "warning": [],
+    "deprecation": []
```

Comment:
Same question here, did you use the nonsplit way of generating?
Never manually create those files!